docs/source/release-history.rst

```diff
@@ -7,6 +7,40 @@ In Development
 
 [Add information about changes in your PR here]
 
+**Breaking Changes:**
+
+- The order of parameters in :meth:`wayback.WaybackClient.get_memento` has changed, and additional parameters for the date and mode of memento playback have been added. You can now call ``get_memento()`` with the URL and time you want the memento of, rather than needing to compose a complete Wayback memento URL. That is, you can now call:
```

Comment:
might be helpful to call out "can" vs "must", specifically

> We still support the old behavior above (both for backwards compatibility and because it's useful).

Reply:
I’m not sure I’m following here. Which advice/guidance needs clarification?

Reply:
Listing this change under "breaking changes" makes it read as though it no longer supports the old format – but my read of your description elsewhere is that the old format is still supported. Is that correct? Or are the parameters moved around even in the old format?

Reply:
Ahhhh, so if you used to use it in the most simple way, you’re fine and nothing breaks:

```py
client.get_memento('http://web.archive.org/web/20180816111911id_/http://www.noaa.gov/')
```

And if you used any of the other parameters as keyword arguments instead of positional arguments, you’re still fine:

```py
client.get_memento('http://web.archive.org/web/20180816111911id_/http://www.noaa.gov/',
                   exact=False,
                   exact_redirects=False,
                   target_window=3600)
```

BUT you used to be able to specify all those other things as positional arguments, too, which is no longer supported. So this used to work, but now breaks:

```py
# Same as above, but using positional instead of keyword args. No longer works.
client.get_memento('http://web.archive.org/web/20180816111911id_/http://www.noaa.gov/',
                   False,
                   False,
                   3600)
```

I should probably break up the discussion of the arguments that are now keyword-only (the breaking part) from the *new* arguments (`mode`, `date_`, and new values for `url`). That would be clearer.

Reply:
(I could have not made the above breaking change, but the code would be messier, and we’re about to break more stuff with #2, so this seemed like a good time to be breaking things in the name of code organization. 😄)